mri/reconstruct/utils.py

```diff
@@ -218,18 +228,64 @@ def generate_operators(data, wavelet_name, samples, mu=1e-06, nb_scales=4,
         gradient_op = GradSynthesis2(
             data=data,
             linear_op=linear_op,
-            fourier_op=fourier_op)
+            fourier_op=fourier_op,
+            max_iter=lips_calc_max_iter)
         prox_op = SparseThreshold(Identity(), mu, thresh_type="soft")
     else:
         gradient_op = GradAnalysis2(
             data=data,
-            fourier_op=fourier_op)
+            fourier_op=fourier_op,
+            max_iter=lips_calc_max_iter)
         prox_op = SparseThreshold(linear_op, mu, thresh_type="soft")
-
     # Define the cost function
     # TODO need to have multiple cost functions with a parameter
     cost_op = GenericCost(
         gradient_op=gradient_op,
         prox_op=prox_op,
         verbose=verbose)
     return gradient_op, linear_op, prox_op, cost_op
+
+
+def get_stacks_fourier(kspace_loc):
+    """ Function that converts an incoming 3D kspace samples
```

Comment:
I rather change this sentence for something like:
```suggestion
    """ Function that splits an incoming 3D stacked k-space samples 
    into a 2D non-Cartesian plane and the vector containing the z k-space values 
    of all the plane 
```